accessibility-checker-extension/src/ts/devtools/DevToolsPanelApp.tsx

```diff
@@ -321,7 +321,7 @@ selectPath("${item.path.dom}");
         }
     }
 
-    getItem(item: IReportItem) {
```

Comment:
Why was this change made? This is a member function declaration.

Reply:
changed back